### ISSUE TITLE
D8: Type com.websitebeaver.documentscanner.BuildConfig is defined multiple times

### ISSUE THREAD

**Issue by Newbie012** [WebsiteBeaver/capacitor-document-scanner#3]
While running the following command:

```
./gradlew clean bundleRelease
```

I'm getting the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeDexRelease'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.DexMergingTaskDelegate
   > There was a failure while executing work items
      > A failure occurred while executing com.android.build.gradle.internal.tasks.DexMergingWorkAction
         > com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: 
           Learn how to resolve the issue at https://developer.android.com/studio/build/dependencies#duplicate_classes.
           Type com.websitebeaver.documentscanner.BuildConfig is defined multiple times: /.../node_modules/.pnpm/capacitor-document-scanner@0.2.0_@capacitor+core@3.5.0/node_modules/capacitor-document-scanner/android/build/.transforms/0c5137feee1f7f0dd263bea070ce71e2/transformed/classes/classes.dex, /.../android/app/build/intermediates/external_libs_dex/release/mergeExtDexRelease/classes.dex
```

Looks like it happens in your own [example folder](https://github.com/WebsiteBeaver/capacitor-document-scanner/tree/master/example) as well

**Comment by ElLelero** [comment]
@Newbie012 Facing the same issue here, did you find a solution?

**Comment by ElLelero** [comment]
@Newbie012 If you're still interested, I found a fix.
Add these lines of code at the end of your build.gradle (node_modules/capacitor-document-scanner/android/build):

```
afterEvaluate {
	generateReleaseBuildConfig.enabled = false
	generateDebugBuildConfig.enabled = false
	generateReleaseResValues.enabled = false
	generateDebugResValues.enabled = false
}
```

**Comment by Newbie012** [comment]
@ElLelero Thanks for sharing! seems like a PR or a fork is required then.

**Comment by dmarcs** [comment]
fixed in 3b1a2f4310a2ab8410479a28173f6a25e0962fa8
